wandb/apis/public/api.py

```diff
@@ -1535,15 +1549,14 @@ def integrations(
                 fetch integrations.  If not provided, the user's default entity
                 will be used.
             per_page (int, optional): Number of integrations to fetch per page.
-                Defaults to 50.
+                Defaults to 50.  Usually there is no reason to change this.
```

Comment:
```suggestion
                Defaults to 50, which is appropriate in most cases.
```
If you take this suggestion, there are several similar instances below. If changing it is actually risky, we could go further and say something like "Change this only in collaboration with W&B Support."